components/cluster/resource-agents/resource-agents.p5m

```diff
@@ -0,0 +1,341 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013 Adam Stevko. All rights reserved.
+# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+# make ocf scripts exectuable
+<transform file path=usr/lib/ocf/resource.d/(.+)$  -> default mode 0555 >
+<transform file path=usr/lib/ocf/heartbeat/(.+)$  -> default mode 0555 >
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license resource-agents.gplv2.license license='GPLv2'
+license resource-agents.gplv3.license license='GPLv3'
+license resource-agents.lgpl.license license='LGPLv2'
+
+link path=etc/ha.d/resource.d/ldirectord target=../../../usr/sbin/ldirectord
+link path=usr/lib/ocf/resource.d/redhat  target=../../../share/cluster
+
+file path=etc/ha.d/shellfuncs
+file path=etc/init.d/ldirectord
+file path=etc/logrotate.d/ldirectord
+file path=usr/include/heartbeat/agent_config.h
+file path=usr/lib/heartbeat/findif
```

Comment:
handled in
<transform file path=usr/lib/heartbeat/(.+)$  -> default mode 0555 >